tests/dev.yml

```diff
@@ -8,6 +8,10 @@ parameters:
         source: https://raw.githubusercontent.com/appuio/component-openshift4-operators/v1.4.0/lib/openshift4-operators.libsonnet
         output_path: vendor/lib/openshift4-operators.libsonnet
 
+  components:
+    appcat:
+      version: debug
```

Comment:
This is probably not intentended right?

Reply:
I'm inclined to keep it, otherwise it will default to `master` and we don't see if it actually concatenates correctly.